arcsi/api/utils.py

```diff
@@ -159,6 +159,8 @@ def get_shows():
     return shows
 
 def item_duplications_number(item):
-    name_occurrence = int(db.session.query(db.func.count()).filter(Item.name == item.name, Item.number == item.number).scalar())
+    show_items = Show.query.filter(Show.id == item.shows[0].id).first().items
+    show_items_with_same_name = show_items.filter_by(name=item.name)
+    name_occurrence = show_items_with_same_name.count()
     app.logger.error("Name_occurence (duplicate detection): {}".format(name_occurrence))
-    return name_occurrence
+    return 0 < name_occurrence
```

Comment:
I mean you probably wanted to have a more exact boolean return here and get `True` or `False` instead of `1` and `0` 

But if you think about this adjustment in broad terms it does matter a bit since now you changed the return type from integer to boolean so if it is used somewhere as a number not just a logical statement this change could break the application.

It can be a good change to make it return boolean but on changes like this we either should add inline documentation or more details on the PR description about this.